framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -2955,7 +2955,21 @@ private AnnotationMirror getDeclAnnotation(
 
         Set<AnnotationMirror> results = AnnotationUtils.createAnnotationSet();
         // Retrieving the annotations from the element.
-        results.addAll(elements.getAllAnnotationMirrors(elt));
+        List<? extends AnnotationMirror> fromEle = elements.getAllAnnotationMirrors(elt);
+        for (AnnotationMirror annotation : fromEle) {
+            try {
```

Comment:
Sure. Can you at least file an issue? This seems like an important cleanup that would prevent future CompletionErrors from creeping up in other places.